homeassistant/components/plugwise/climate.py

```diff
@@ -146,82 +149,47 @@ def supported_features(self):
     def device_state_attributes(self):
         """Return the device specific state attributes."""
         attributes = {}
-        if self._outdoor_temperature:
-            attributes["outdoor_temperature"] = self._outdoor_temperature
         if self._schema_names:
-            attributes["available_schemas"] = self._schema_names
+            if len(self._schema_names) > 1:
+                attributes["available_schemas"] = self._schema_names
         if self._selected_schema:
             attributes["selected_schema"] = self._selected_schema
-        if self._boiler_temperature:
-            attributes["boiler_temperature"] = self._boiler_temperature
-        if self._water_pressure:
-            attributes["water_pressure"] = self._water_pressure
         return attributes
 
     @property
     def preset_modes(self):
-        """Return the available preset modes list.
-
-        And make the presets with their temperatures available.
-        """
+        """Return the available preset modes list."""
         return self._presets_list
 
     @property
     def hvac_modes(self):
         """Return the available hvac modes list."""
-        if self._heating_status is not None or self._boiler_status is not None:
-            if self._cooling_status is not None:
+        if self._heating_state is not None or self._boiler_state is not None:
+            if self._cooling_state is not None:
                 return HVAC_MODES_2
```

Comment:
Existing, but this constant could use a better name as it doesn't show intent

Reply:
Intent as being `self._is_heating` and `..is_cooling`? If that is what you mean we'll come up with something for boiler.

Reply:
Naming is hard so feel free to adjust:

Suggested:

`HVAC_MODES_2` -> `HVAC_MODES_WITH_HEAT_COOL`
`HVAC_MODES_1` -> `HVAC_MODES_HEAT_ONLY`

Reply:
Good suggestions, I changed the constants accordingly (sorry, misreading the word)